toolkit/README.md

```diff
@@ -0,0 +1,67 @@
+- [Write Codeblocks](#write-codeblocks)
+  - [Usage](#usage)
+- [Check CLI Output](#check-cli-output)
+  - [Usage](#usage-1)
+
+# Write Codeblocks
+
+The `update_spin_cli_reference.py` script will fetch the current Spin CLI reference document from the web and write a fresh copy of the file (with the new version of code blocks added) to your local disk. This new markdown is the start of a PR that contains CLI commands for the new version of Spin.
+
+## Usage
+
+Edit the version numbers in the file i.e. `2.4`, `2.5` etc. and run the script:
+
+```bash
+python3 update_spin_cli_reference.py
+```
+
+The following `check_cli_output.py` script will tell you what the changes are (between versions) these scripts are designed to quickly kick off the edit, review, approve, and merge process at the time of release.
+
+# Check CLI Output
+
+The `check_cli_output.py` script will run `spin --help` for all subcommands recursively. This is useful for quickly detecting changes in the Spin CLI's output between different versions (i.e. when updating the Spin CLI Reference page in the documentation during each release).
+
+## Usage
+
+We:
+- create a variable for each version
+- create two new directories
+- copy the script into each directory (this makes the script's output identical - aside from actual changes)
+- install the specific version of Spin in each directory
+- run the script using that specific version
+- diff the results
+
+```bash
```

Comment:
Do we want to include this bash script (or some form of it) to make it even easier for folks to run it all?  eg user might then just need to run `check-cli-diff.sh v2.5.0 v2.6.0` (the script or user may need to do some python setup...)

Reply:
Thanks @vdice - that's a great idea - we now have a bash script that takes two arguments, does the whole job and the clean up.